sss/tests/test_city.py

```diff
@@ -21,6 +22,21 @@ def test_add_city():
 
     assert list(test_df["state"]) == ["AZ", "KS", "OR", "PA"]
 
+    with pytest.raises(
+        ValueError, match="could not find state in State value"
+    ) as exc_info:
+        add_city(
+            os.path.join(
+                DATA_PATH,
+                "city_data",
+                "2020_PopulationDatabyCity_20220804_Ama_incorrect_state_name.xlsx",
+            ),
+            2021,
+        )
+
+    assert exc_info.type is ValueError
+    assert "could not find state in State value" in exc_info.value.args[0]
```

Comment:
You don't need these lines. The pytest.raises checks that the error type is what you specify and that the error message matches what you pass to `match`.